feature-libs/storefinder/styles/components/_store-finder-list-item.scss

```diff
@@ -36,6 +36,9 @@
     @include media-breakpoint-down(md) {
       white-space: normal;
     }
+    @include forFeature('a11yTruncatedTextStoreFinder') {
+      white-space: normal;
+    }
```

Comment:
Looks good! Small improvement we could make is to put this change below the property it overwrites and use `white-space: unset`. Since ideally we would remove the `white-space: nowrap` and use the default value. This should make it clearer to the dev that will remove this FF later that this is what we intended .